package.json

```
@@ -29,16 +29,15 @@
   "author": "Isaac Z. Schlueter <i@izs.me> (http://blog.izs.me)",
   "repository": {
     "type": "git",
-    "url": "https://github.com/npm/cli"
+    "url": "git+https://github.com/npm/cli.git"
```

```suggestion
    "url": "https://github.com/npm/cli.git"
```

this would match with how we generally do it elsewhere

---

Template-oss is normalizing these now https://github.com/npm/template-oss/blob/755a300568d2ec1aca41f791c2dd88185be323c6/lib/config.js#L158-L164